lib/matplotlib/__init__.py

```diff
@@ -1184,13 +1184,13 @@ def __exit__(self, exc_type, exc_value, exc_tb):
 @cbook._rename_parameter("3.1", "arg", "backend")
 def use(backend, warn=False, force=True):
     """
-    Set the matplotlib backend to one of the known backends.
+    Select the matplotlib backend for rendering.
```

Comment:
not just for rendering -- also for GUI toolkit integration (in fact, mostly for that purpose, as I'd guess most uses are selecting qt5agg vs gtk3agg vs tkagg rather than agg vs cairo vs pdf...)

Reply:
The interpretation of "rendering" should be broad here, as in "Render into a Qt window". The original docstring was more or less tautological. OTOH just "Set the matplotlib backend." is a bit short and does not help if you have no clue what a backend is for.

Better descriptions are welcome.

Reply:
"Select the backend used for rendering and GUI integration."?